src/services/dashboards/widgets/budget-usage-summary/BudgetUsageSummaryWidget.vue

```diff
@@ -17,13 +18,13 @@
             </div>
             <div class="budget">
                 <p class="budget-label">
-                    {{ $t('DASHBOARDS.WIDGET.BUDGET_USAGE_SUMMARY.TOTAL_SPENT') }}
+                    {{ $t('DASHBOARDS.WIDGET.BUDGET_USAGE_SUMMARY.TOTAL_BUDGET') }}
                 </p>
                 <div class="budget-value">
                     {{ currencyMoneyFormatter(state.totalBudget, state.options.currency) }}
                 </div>
                 <div class="budget-info">
-                    {{ currencyMoneyFormatter(state.leftBudgetRate, state.options.currency) }} {{ $t('DASHBOARDS.WIDGET.BUDGET_USAGE_SUMMARY.AVAILABLE') }}
+                    {{ currencyMoneyFormatter(state.leftBudget, state.options.currency) }} {{ $t('DASHBOARDS.WIDGET.BUDGET_USAGE_SUMMARY.AVAILABLE') }}
```

Comment:
suggestion: What about using `currencyMoneyFormatter` and adding `$t('DASHBOARDS.WIDGET.BUDGET_USAGE_SUMMARY.AVAILABLE')` at `state.leftBudget` not to add `$t('DASHBOARDS.WIDGET.BUDGET_USAGE_SUMMARY.AVAILABLE')` in no data case?

```suggestion
                    {{ state.leftBudget }}
```

```typescript
    leftBudget: computed(() => {
        if (!state.data) return '--';
        const value = state.totalBudget - state.totalSpent;
        return `${currencyMoneyFormatter(value, state.options.currency)} ${i18n.t('DASHBOARDS.WIDGET.BUDGET_USAGE_SUMMARY.AVAILABLE')}` ;
    }),
```